meson.build

```diff
@@ -108,23 +108,35 @@ if not get_option('source-only')
     sdl_options += 'use_video=enabled'
     sdl_options += 'use_atomic=enabled'
     sdl_options += 'use_threads=enabled'
+    sdl_options += 'use_timers=enabled'
     # investigate if this is truly needed
     # Do not remove before https://github.com/libsdl-org/SDL/issues/5413 is released
     sdl_options += 'use_events=enabled'
 
-    # we leave this up to what the host system has
-    sdl_options += 'use_video_x11=auto'
-    sdl_options += 'use_video_wayland=auto'
+    if host_machine.system() == 'darwin' or host_machine.system() == 'windows'
+        sdl_options += 'use_video_x11=disabled'
+        sdl_options += 'use_video_wayland=disabled'
```

Comment:
I wonder if this is a wrap bug. I'd expect that those would be automatically disabled on unsupported platforms.

Reply:
MacOS supported X11 and there is still some xserver packages floating for it, also it seems that the github runner does have those packages installed so x11 support gets enabled and linked in, the issue is that the user most of the time will not normally have that library installed so we have to explicitly disable it at least for darwin, did it for windows too just in case...

Reply:
Fair enough.